README.md

```diff
@@ -329,6 +329,41 @@ sample_and_identify_pipeline --project="${PROJECT_ID}" \
 --reportLocation="gs://${TEMP_GCS_BUCKET}/dlp_report/"
 ```
 
+### Launch sample & identify pipeline from Python
+
+First, build the Python module:
+
+```shell script
+./gradlew pythonDist -x test
+```
+
+Add the Python module (zip-file) to Python's library path
+
+```Python
+import sys
+sys.path.append('dapla-dlp.zip')
+```
+
+And then execute the sample & identify pipeline through Python
+
+```Python
+import os
+from dapla_dlp import PipelineOptions, start_dlp_inspection_pipeline
+
+options = PipelineOptions(
+    projectId = os.environ['PROJECT_ID'],
+    regionId = os.environ['REGION_ID'],
+    serviceAccount = os.environ['DLP_RUNNER_SERVICE_ACCOUNT_EMAIL'],
+    tempGcsBucket = os.environ['TEMP_GCS_BUCKET'],
+    vpcName = os.environ['VPC_NAME'],
+    inputPattern = f"gs://{os.environ['TEMP_GCS_BUCKET']}/userdata.avro",
```

Comment:
Maybe we should introduce two different buckets.. one for holding source data and one for the temporary and dlp report files?